daml/Marketplace/Trading/Model.daml

```diff
@@ -93,3 +95,32 @@ template Order
     key (provider, details.id.label) : (Party, Text)
     maintainer key._1
 
+data Fee = Fee with
+    amount : Decimal
+    currency : Id
+    timeInEffect : Time
+  deriving (Show, Eq)
```

Comment:
This is the `Fee` per trade ?

Reply:
This is the fee per order. As the order is filled, the fee is split and taken by the Exchange until the order is completely filled.